docs/metachromium-overview.md

```diff
@@ -3,5 +3,54 @@ id: metachromium-overview
 title: Metachromium Overview
 ---
 
-Metachromium
+[Metachromium](https://metachromium.com/) is a spatial app browser that runs side-by side with OpenVR. It lets you open any WebXR page as a spatial overlay app that runs on top of any other VR game.
 
+## Features
+
+- Run WebXR context on top of an OpenVR application
+
+- Run multiple WebXR applications inside a single Metachromium context
+  
+- Add multiplayer to singleplayer apps using metaRTC
```

Comment:
MetaRTC is not a feature of Metachromium, it is distributed as a normal XRPackage that can be added to your scene. MetaRTC needs to be covered in a dedicated page.